src/containers/ManageListingsPage/ManageListingCard/ManageListingCard.js

```diff
@@ -127,7 +127,7 @@ const ShowFinishDraftOverlayMaybe = props => {
       >
         <NamedLink
           className={css.finishListingDraftLink}
-          name="EditListingPage"
```

Comment:
Let's talk about this later today. If it's ok to completely overwrite the link or make it conditional